match/lib/match/runner.rb

```diff
@@ -31,6 +31,12 @@ def run(params)
       TablePrinter.print_summary(params, uuid)
 
       UI.success "All required keys, certificates and provisioning profiles are installed 🙌".green
+    rescue Spaceship::Client::UnexpectedResponse, Spaceship::Client::InvalidUserCredentialsError, Spaceship::Client::NoUserCredentialsError => ex
+      UI.error("An error occured while verifying your certificates and profiles with the Apple Developer Portal")
```

Comment:
`UI.error("An error occured while verifying your certificates and profiles with the Apple Developer Portal.")`